src/entrypoint.sh

```diff
@@ -4,7 +4,7 @@ set -o nounset
 set -o errexit
 # Sha-bang cannot be /bin/bash (not available), but
 # the container's /bin/sh does support pipefail.
-# shellcheck disable=SC2039
+# shellcheck disable=SC2039,SC3040
```

Comment:
What do you think about adding [these links](https://github.com/cisagov/certboto-docker/pull/22#discussion_r632088000) to this comment?

Reply:
Good suggestion.  Added in https://github.com/cisagov/certboto-docker/pull/22/commits/37276f7a78f38a9cff62dba5c9a410b08943161e